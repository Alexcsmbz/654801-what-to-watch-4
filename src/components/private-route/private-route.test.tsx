import PrivateRoute from './private-route';
import {BrowserRouter as Router} from 'react-router-dom';

describe(`PrivateRoute snapshot test`, () => {
  it(`PrivateRoute should render private route component`, () => {
    const tree = renderer.create(
        <Router>
          <PrivateRoute isAuth={false} />
        </Router>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
