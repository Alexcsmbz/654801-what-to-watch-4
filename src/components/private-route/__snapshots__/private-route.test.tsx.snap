// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivateRoute snapshot test PrivateRoute should render private route component 1`] = `null`;
